templates/repo/actions/list.tmpl

```diff
@@ -19,6 +19,50 @@
 				</div>
 			</div>
 			<div class="twelve wide column content">
+				<div class="ui secondary filter stackable menu gt-je">
+					<!-- Actor -->
+					<div class="ui {{if not .Actors}}disabled{{end}} dropdown jump item">
+						<span class="text">
+							{{.locale.Tr "actions.runs.actor"}}
+							{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+						</span>
+						<div class="menu">
+							<div class="ui icon search input">
+								<i class="icon gt-df gt-ac gt-jc">{{svg "octicon-search" 16}}</i>
```

Comment:
I guess `gt-df gt-ac gt-jc` is a no-op, at least #25163 doesn't need them, or we can review and merge #25163 first

Reply:
Yes, with just one SVG child, this is noop.

Reply:
Updated in https://github.com/go-gitea/gitea/pull/25118/commits/738df8f18f8515a2c3c82483300e4570d9307e57